test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/SubmitMessagesForReconnect.java

```diff
@@ -54,15 +54,20 @@ public static void main(String... args) {
 		);
 	}
 
+	private static HapiSpecOperation submitToTestTopic(PerfTestLoadSettings settings) {
+		String topicToSubmit = String.format("0.0.%d",	settings.getTestTopicId());
+		return submitMessageTo(topicToSubmit)
+				.message(randomUtf8Bytes(100))
+				.noLogging()
+				.hasRetryPrecheckFrom(BUSY, DUPLICATE_TRANSACTION, PLATFORM_TRANSACTION_NOT_CREATED)
```

Comment:
DUPLICATE_TRANSACTION should be removed for retries

Reply:
Done